build-farm/sign-releases.sh

```diff
@@ -44,7 +48,7 @@ do
       echo "signing ${file}"
 
       # shellcheck disable=SC2086
-      bash "${SCRIPT_DIR}/../sign.sh" "${file}"
+      bash "${SCRIPT_DIR}/../sign.sh" "${file}" ${CERTIFICATE}
```

Comment:
The arguments should be passed in the same order as upstream to avoid an unnecessary difference.

Reply:
Good point, thanks for catching that Keith. @vsebe can you do another PR at some point?